src/core/core.scale.js

```diff
@@ -850,7 +850,13 @@ module.exports = Element.extend({
 				context.translate(itemToDraw.labelX, itemToDraw.labelY);
 				context.rotate(itemToDraw.rotation);
 				context.font = itemToDraw.major ? majorTickFont.font : tickFont.font;
-				context.fillStyle = itemToDraw.major ? majorTickFontColor : tickFontColor;
+				if (Object.prototype.toString.call(tickFontColor) === '[object Array]') {
```

Comment:
you could call `helpers.isArray` here

Reply:
@benmccann Only if you want more concise and better code. :). Updated.